sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala

```diff
@@ -184,6 +184,9 @@ private[jdbc] class JDBCRDD(
     options: JDBCOptions)
   extends RDD[InternalRow](sc, Nil) {
 
+  import scala.collection.JavaConverters._
```

Comment:
How about moving this into the header and removing the other import?: https://github.com/apache/spark/blob/fd899d6331f4777a36a3f2a79a6b2fa123dccc1a/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRDD.scala#L274